pkgs/applications/networking/avalanchego/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, stdenv
+, IOKit
+}:
+
+buildGoModule rec {
+  pname = "avalanchego";
+  version = "1.9.5";
+
+  src = fetchFromGitHub {
+    owner = "ava-labs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Uim2Q3LvNEV0C5/gFdAiXKy2oKbhhc/ifRyz5BIh6zE=";
+  };
+buildInputs = lib.optionals stdenv.isDarwin [ IOKit ];
+
```

Comment:
```suggestion
# patch zondax/hid dependency manually on darwin until https://github.com/ava-labs/avalanchego/pull/1194 is merged
  patches = [ ./update-zondax-hid.patch ];
```
Here is the patch:
[update-zondax-hid.patch](https://github.com/NixOS/nixpkgs/files/10361315/update-zondax-hid.patch)

Reply:
went with `fetchpatch` so we do not need to check in the file